src/kernel-install/50-depmod.install

```diff
@@ -1,16 +1,28 @@
-#!/bin/bash
+#!/bin/sh
 # -*- mode: shell-script; indent-tabs-mode: nil; sh-basic-offset: 4; -*-
 # ex: ts=8 sw=4 sts=4 et filetype=sh
 
-[[ $2 ]] || exit 1
+[ -n "$2" ] || exit 1
 
 case "$1" in
     add)
-        [[ -d /lib/modules/"$2"/kernel ]] || exit 0
+        [ -d "/lib/modules/$2/kernel" ] || exit 0
         exec depmod -a "$2"
         ;;
     remove)
-        exec rm -f /lib/modules/"$2"/modules.{alias{,.bin},builtin.bin,dep{,.bin},devname,softdep,symbols{,.bin}}
+        for d in
+            alias \
+            alias.bin \
+            builtin.bin \
+            dep \
+            dep.bin \
+            devname \
+            softdep \
+            symbols \
+            symbols.bin
+        do
+            rm -f "/lib/modules/$2/modules.$d"
+        done
```

Comment:
The removal of brace expansion is OK, but please don't make this into a loop, i.e. keep multi-arg `rm` invocation.

Reply:
Okay, will take the note.

Reply:
Can we haz an update here?